packages/bundler-plugin-core/package.json

```diff
@@ -10,14 +10,9 @@
     "access": "public"
   },
   "files": [
-    "dist"
+    "dist",
+    "sentry-release-injection-file.js"
   ],
-  "exports": {
-    ".": {
-      "import": "./dist/esm/index.mjs",
-      "require": "./dist/cjs/index.js"
-    }
-  },
```

Comment:
This was causing weird problems when importing `@sentry/bundler-plugin-core/sentry-release-injection-file`. This standard is cursed.

Reply:
It might be that with this you have to name it `sentry-release-injection-file.mjs` for esm.